DigitalLearningSolutions.Web/Models/Enums/UserCard.cs

```diff
@@ -1,6 +1,6 @@
 ﻿namespace DigitalLearningSolutions.Web.Models.Enums
 {
-    enum UserCard
+    public enum UserCard
```

Comment:
I think this is the only change required to make the Super Admin work again. I tried this in my local and it works. The certificates however are still not working in my local. Let me try to figure a fix for certificates as well, and then we can push another commit in the same PR.